### PR TITLE
fix(flexbox): normalize flexbox mixins and usage

### PR DESCRIPTION
**Pull request by CITguy**
* pulled in better flexbox mixins from https://github.com/CITguy/moar-elements/blob/master/flexbox.less
  * majority of mixins match non-prefixed properties
* replaced prefixed properties with mixins
* Modified existing mixins to use new mixin definitions
  * .flexbox-direction() -> .flex-direction()
  * .flexbox-order() -> .order()
  * .flexbox-display() -> .flexbox()
  * .flexbox-wrap() -> .flex-wrap()
  * .flexbox-pack() -> .justify-content()
  * .flexbox-align() -> .align-items()
  * .flexbox-grow() -> .flex-grow()
  * .flexbox() -> .flex()

### CODE REVIEW

File: src/styles/flexbox.less [hunk 1]
```diff
@@ -1,62 +1,121 @@
...
+     -moz-align-content: @arguments;
+          align-content: @arguments;
+}//.align-content()
+
+
+.align-self(...) {
+  -webkit-align-self: @arguments;
+     -moz-align-self: @arguments;
+          align-self: @arguments;
+}//.align-self()
```
Comment (Droogans) [inline src/styles/flexbox.less]: These definitions are the only ones that include a block-terminating comment. I think it should look like the rest of the less code and not include these `//.align-self()`, etc. comments.

**Comment by CITguy** [comment]
@Droogans: removed block-terminating comments

**Comment by parlarjb** [comment]
Have you tested this against Cloud or TQ, to see if it causes any visual issues?

**Comment by CITguy** [comment]
Checked against Cloud UI in Chrome and Firefox. Made a tweak, but everything seems to look and behave correctly.

**Comment by CITguy** [comment]
Title changed to reflect "billing-only" moratorium.

**Comment by CITguy** [comment]
Rebased, awaiting Travis results.

**Comment by CITguy** [comment]
Rebase has no conflicts with master. Travis gave his ok. Good to merge.

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**